test/utils/wabt_engine.cpp

```diff
@@ -76,14 +84,6 @@ std::optional<WasmEngine::FuncRef> WabtEngine::find_function(std::string_view na
 WasmEngine::Result WabtEngine::execute(
     WasmEngine::FuncRef func_ref, const std::vector<uint64_t>& args)
 {
-    // WABT Executor/Thread will init both value and call stack vectors with zeros up front.
-    // To mitigate the performance overhead, we set sizes to much lower values
```

Comment:
Why do we not do this anymore?

Reply:
Because we create single executor per `WabtEngine` instance and the initialization cost of it is not measured any more. So we can use default values as it does not matter any more.